### ISSUE TITLE
Search Player Chemistry?

### ISSUE THREAD

**Issue by googlme123** [hunterjm/fifa-autobuyer#106]
Is there an option for Player Chemistry to Search that Also?

**Comment by frayzee** [comment]
No. However an option to search for a select Chemisty style of a select position on a playet would improve the coin earning potential of this AB 1000%. I'm not sure how difficult to impliment this would be as Hunter has already done fantastic work we can't complain.
